src/index.js

```diff
@@ -54,25 +54,81 @@ const metrics = {
 		name: 'pagespeed_accessibility_score',
 		help: "Accessibility Score",
 		labelNames: [ 'page' ]
+	}),
+	mobile_first_contentful_paint: new Prometheus.Gauge({
+		name: 'pagespeed_mobile_first_contentful_paint_milliseconds',
+		help: 'Time to first contentful paint, more info here: https://web.dev/first-contentful-paint/',
+		labelNames: [ 'page' ]
+	}),
+	mobile_first_cpu_idle: new Prometheus.Gauge({
+		name: 'pagespeed_mobile_first_cpu_idle_milliseconds',
+		help: "First CPU Idle marks the first time at which the page's main thread is quiet enough to handle input.  https://web.dev/first-cpu-idle.",
+		labelNames: [ 'page' ]
+	}),
+	mobile_interactive: new Prometheus.Gauge({
+		name: 'pagespeed_mobile_interactive_milliseconds',
+		help: "Time to interactive is the amount of time it takes for the page to become fully interactive. https://web.dev/interactive.",
+		labelNames: [ 'page' ]
+	}),
+	mobile_speed_index: new Prometheus.Gauge({
+		name: 'pagespeed_mobile_speed_index_seconds',
+		help: "Speed Index, more info here: https://web.dev/speed-index",
+		labelNames: [ 'page' ]
+	}),
+	mobile_max_potential_fid: new Prometheus.Gauge({
+		name: 'pagespeed_mobile_max_potential_fid_seconds',
+		help: "The maximum potential First Input Delay that your users could experience is the duration, in milliseconds, of the longest task. https://developers.google.com/web/updates/2018/05/first-input-delay",
+		labelNames: [ 'page' ]
+	}),
+	mobile_first_meaningful_paint: new Prometheus.Gauge({
+		name: 'pagespeed_mobile_first_meaningful_paint_seconds',
+		help: "First Meaningful Paint measures when the primary content of a page is visible. https://web.dev/first-meaningful-paint",
+		labelNames: [ 'page' ]
+	}),
+	mobile_performance_score: new Prometheus.Gauge({
+		name: 'pagespeed_mobile_performance_score',
+		help: "Performance Score",
+		labelNames: [ 'page' ]
+	}),
+	mobile_accessibility_score: new Prometheus.Gauge({
+		name: 'pagespeed_mobile_accessibility_score',
+		help: "Accessibility Score",
+		labelNames: [ 'page' ]
 	})
 };
 
 for (let page of pages) {
 	setInterval(async () => {
-		let data = await pagespeed(page, apiKey);
-		if (data != null) {
+		let desktopData = await pagespeed(page, apiKey, 'DESKTOP');
```

Comment:
Have you thought about using constants instead of strings to decouple the internal implementation?

Reply:
Wasn't 100% sure on what you meant by this. Have just committed something, not sure if that's the kind of thing you were thinking of.